input/pagecontent/sushi.md

```diff
@@ -0,0 +1,130 @@
+
+## Installation
+Install SUSHI in two easy steps:
+
+1) Install Node.js
+    
+    a) Download from https://nodejs.org 
+    
+    b) Run the installer
+
+    c) Check the installation by opening a command window and typing the following two commands. Each command should return a version number. 
+
+    `$ node --version`
+
+    `$ npm --version`
+
+1) Install SUSHI
+
+    a) Issue this command at the prompt:
+
+    `$ npm install -g fsh-sushi`
+
+    b) Check the installation by typing the following command:
+
+    `$ sushi -h`
+
+If the command outputs instructions on using SUSHI command line interface (CLI), you're ready to run SUSHI.
+
+## SUSHI Version
+
+SUSHI follows the [semantic versioning](https://semver.org) convention (MAJOR.MINOR.PATCH):
+
+* MAJOR: A major release has significant new functionality and potentially, grammar changes or other non-backward-compatible changes.
+* MINOR: Contains new or modified features, while maintaining backwards compatibility within the major version.
+* PATCH: Contains minor updates and bug fixes, while maintaining backwards compatibility within the major version.
+
+_Is this correct?_
+
+> The config file must declare the minimum version of SUSHI compiler it requires. For example, if the config declares SUSHI 1.2, it requires major version 1, any minor version >= 2.
+
+>**NOTE:** There is no guarantee that a FSH file declaring major version X will compile correctly on major version Y, if X /= Y.
+
+_end_
+
+## Configuration File
+
+Shorthand uses the FHIR packaging approach:
+
+* packages are defined using a package.json
+* dependencies are defined in the package.json
+* package managers handle the resolution of dependency packages
+* dependency resolution follows the same rules as FHIR dependency resolution
+
+> **NOTE**: In addition, however, we may want to support dependencies for which there is not a _built_ package yet, but there is an accessible GitHub repository or even a local folder representing a package (NPM supports this).
+
+See: [FHIR NPM Package Spec](https://wiki.hl7.org/index.php?title=FHIR_NPM_Package_Spec#Format)
+
+
+## Executing SUSHI from Command Line
+
+The general form of the SUSHI execution command is as follows:
+
+`$ sushi {specification-directory} {options}`
+
+where options include:
+
+```
+-o, --out <out>          the path to the output folder (default: /out)
+-h, --help               output usage information
+```
+
+The options are not order-sensitive.
+
+If you run SUSHI from the same folder where your .fsh files are located, the command can be shortened to:
+
+`$ sushi . {options}`
+
+
+## Error Messages
+
+In the process of building profiles, you will inevitably encounter SUSHI error messages. Debugging the model is an iterative process, and it could take some time to arrive at a clean compile. SUSHI will only produce IG artifacts if it runs to completion without errors.
```

Comment:
This isn't totally true. If one profile had errors, it wouldn't stop the other profiles from compiling into SDs. Generally we try to go on if we can, so if a specific rule is an error, we would log an error, ignore that rule, and still try to make an SD using the other rules on that profile.